database_backup/.metadata/metadata.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Database backup",
+  "description": "A tool that runs a database backup and uploads the backup to AWS S3.",
+  "tags": ["type:Task", "task:Shell"],
+  "tasks": [
+    {
+      "name": "Database backup",
+      "fileName": "/database_backup.sh"
+    }
+  ],
+  "visualDescription": "This tool allows a user to specifiy a name for the backup file to be generated. It then runs, creates the backup and uploads the resulting .zip file to AWS S3.",
+  "docFeatures": [
+    {
+      "name": "Docker-based environments",
```

eh? Just not to confuse "environments" with the product feature

```suggestion
      "name": "Custom Dockerfile",
```

---

👍 Good call